jaxrs/WFLY-13122_EAP7-1494_Add_Json_Merge_Patch_support.adoc

```diff
@@ -0,0 +1,138 @@
+---
+categories:
+  - jaxrs
+stability-level: default
+issue: https://github.com/wildfly/wildfly-proposals/issues/664
+feature-team:
+ developer: ema
+ sme:
+  - jamezp
+  - ronsigal
+  - liweinan
+ outside-perspective:
+  - honza-kasik
+promotes:
+promoted-by:
+---
+= Add JSON Merge Patch support
+:author:            Jim Ma
+:email:             ema@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          json merge patch,resteasy,jaxrs
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse
+
+== Overview
+This feature is to support JSON Merge Patch spec as described in: https://tools.ietf.org/html/rfc7386
```

Comment:
Sorry I didn't catch this before. The RFC link should probably be https://datatracker.ietf.org/doc/html/rfc7396. RFC 7386 is obsolete and replaced by RFC 7396. We should already be implementing it correctly.

Reply:
I added this link years ago and it's obsoleted.Fixed.